### ISSUE TITLE
Errors on running as a Spring boot application

### ISSUE THREAD

**Issue by jerin-fdo** [ramswaroop/jbot#82]
```java
on running as a spring boot application after changing the Application.properties
2017-11-30 15:30:25.115  INFO 10764 --- [           main] o.s.boot.SpringApplication               : Starting SpringApplication v1.4.0.RELEASE on BDC7-L-C4BXLV1 with PID 10764 (C:\Users\maria.jerin.jesudoss\.m2\repository\org\springframework\boot\spring-boot\1.4.0.RELEASE\spring-boot-1.4.0.RELEASE.jar started by maria.jerin.jesudoss in C:\Users\maria.jerin.jesudoss\Downloads\jbot-master\jbot-master\jbot)
2017-11-30 15:30:25.130  INFO 10764 --- [           main] o.s.boot.SpringApplication               : No active profile set, falling back to default profiles: default
2017-11-30 15:30:25.130  WARN 10764 --- [           main] o.s.boot.SpringApplication               : Error handling failed (ApplicationEventMulticaster not initialized - call 'refresh' before multicasting events via the context: org.springframework.boot.context.embedded.AnnotationConfigEmbeddedWebApplicationContext@70e8f8e: startup date [Thu Jan 01 05:30:00 IST 1970]; root of context hierarchy)
2017-11-30 15:30:25.615 ERROR 10764 --- [           main] o.s.boot.SpringApplication               : Application startup failed

java.lang.IllegalArgumentException: Sources must not be empty
	at org.springframework.util.Assert.notEmpty(Assert.java:276) ~[spring-core-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:363) [spring-boot-1.4.0.RELEASE.jar:1.4.0.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:311) [spring-boot-1.4.0.RELEASE.jar:1.4.0.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1185) [spring-boot-1.4.0.RELEASE.jar:1.4.0.RELEASE]
	at org.springframework.boot.SpringApplication.main(SpringApplication.java:1201) [spring-boot-1.4.0.RELEASE.jar:1.4.0.RELEASE]
```

**Comment by jerin-fdo** [comment]
java.lang.IllegalArgumentException: Sources must not be empty
	at org.springframework.util.Assert.notEmpty(Assert.java:276) ~[spring-core-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:363) [spring-boot-1.4.0.RELEASE.jar:1.4.0.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:311) [spring-boot-1.4.0.RELEASE.jar:1.4.0.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1185) [spring-boot-1.4.0.RELEASE.jar:1.4.0.RELEASE]
	at org.springframework.boot.SpringApplication.main(SpringApplication.java:1201) [spring-boot-1.4.0.RELEASE.jar:1.4.0.RELEASE]

**Comment by rampatra** [comment]
How are you guys running the application? Have you read the documentation?

**Comment by Sjha2885** [comment]
Could you please share the steps to run.

I run as a java application and spring boot application .

**Comment by oyedotunsodiq045** [comment]
> How are you guys running the application? Have you read the documentation?

Running it as a spring boot application

**Comment by MarcelWilnicki** [comment]
perhaphs oyu are running the wrong java class? Go to class where you have your @SpringBootApplication and right click -> run as java application. Hopefully that helps. Cheers

**Comment by rampatra** [comment]
Please follow the instructions under "Getting Started" section of the README: https://github.com/rampatra/jbot/blob/master/docs/slack/README-Slack-JBot-4.0.0.md#getting-started

TL;DR

```bash
$ git clone git@github.com:ramswaroop/jbot.git
$ cd jbot
$ mvn clean install
$ cd jbot-example
$ mvn spring-boot:run
```
